tracing-subscriber/Cargo.toml

```diff
@@ -36,7 +36,7 @@ tracing-core = "0.1.2"
 # only required by the filter feature
 matchers = { optional = true, version = "0.0.1" }
 regex = { optional = true, version = "1" }
-smallvec = { optional = true, version = "0.6.10"}
+smallvec = { optional = true, version = "1.1.0"}
```

Comment:
Do you think this should be more permissive (i.e., just "1") to avoid dep duplication in crates depending on 1.0.x? Do we need any APIs added in 1.1.x?

Reply:
We clearly don't, fair point! I've updated it to require "1".

Reply:
great, looks good to me! merging now!